wasm/store.go

```diff
@@ -314,6 +315,10 @@ func (s *Store) Instantiate(module *Module, name string) error {
 }
 
 func (s *Store) CallFunction(moduleName, funcName string, params ...uint64) (results []uint64, resultTypes []ValueType, err error) {
```

Comment:
I think we can delete this option and then don't have to name it differently.

ex. dapr where there's no option except with context, it is simpler naming 

https://github.com/dapr/go-sdk/blob/main/client/client.go

Since this is an early stage, we can break api now

Reply:
Did you know that I work on Dapr or was this coincidence? 😀

Reply:
I did notice 😎 so decided to speak Lingua Dapra